apps/api/v2/src/ee/event-types/event-types_2024_06_14/outputs/get-event-types.output.ts

```diff
@@ -0,0 +1,17 @@
+import { ApiProperty } from "@nestjs/swagger";
+import { Type } from "class-transformer";
+import { IsArray, IsEnum, ValidateNested } from "class-validator";
+
+import { SUCCESS_STATUS, ERROR_STATUS } from "@calcom/platform-constants";
+import { EventTypeOutput_2024_06_14 } from "@calcom/platform-types";
+
+export class GetEventTypesOutput_2024_06_14 {
+  @ApiProperty({ example: SUCCESS_STATUS, enum: [SUCCESS_STATUS, ERROR_STATUS] })
+  @IsEnum([SUCCESS_STATUS, ERROR_STATUS])
```

Comment:
same herre

Reply:
Refactored in [refactor: use IsIn instead of IsEnum](https://github.com/calcom/cal.com/pull/15457/commits/5ac9ed5ed4a7b35de81b29244a7706616f3f0334)